build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,275 @@
+        if (milestone) {
+          if (milestoneTitle.startsWith('Sprint') && milestoneState == 'closed') {
+            // 4 is the number for Milestone 'Backlog Bugs' 
+            issueNewMilestone = 4; 
+            updates.push(applyMilestone(issue, issueNewMilestone));
+          }
+        }
+        //if issueType is not null, add correct milestones
+        if (issueType != null) {
+          if (milestoneTitle == 'Pending Triage' || milestone == null) {
+            if (issueType == 'Type: Feature Request') {
+              // 23 is the number for Milestone 'New FRs' 
+              issueNewMilestone = 23;
+              updates.push(applyMilestone(issue, issueNewMilestone)); 
+            } else {
+              if (issueType == 'Related to: Documentation' || 
+                issueType == 'Type: Design Review' || issueType == 'Type: Weekly Status') {
+                // 12 is the number for Milestone 'Docs Updates' 
+                issueNewMilestone = 12;
+                updates.push(applyMilestone(issue, issueNewMilestone));
+              } else {
+                if (issueType == 'Type: Bug' || issueType == 'Related to: Flaky Tests') {
+                  // 4 is the number for Milestone 'Backlog Bugs' 
+                  issueNewMilestone = 4;
+                  updates.push(applyMilestone(issue, issueNewMilestone));
+                } else {
+                  if (milestone == null) {
+                    // 20 is the number for Milestone 'Pending Triage' 
+                    updates.push(applyMilestone(issue, issueNewMilestone));
+                  }
+                }
+              }
+            }
+          }
+        } else {
+          if (milestone == null) {
+            // 20 is the number for Milestone 'Pending Triage' 
+            updates.push(applyMilestone(issue, issueNewMilestone));     
+          } else {
+            if (milestoneTitle == 'Prioritized FRs' || milestoneTitle == 'New FRs') {
+              updates.push(applyLabel(issue, 'Type: Feature Request'));
+            } else {
+              if (milestoneTitle == 'Backlog Bugs' || milestoneTitle.startsWith('Sprint')) {
+                updates.push(applyLabel(issue, 'Type: Bug'));
+              }
+            } 
+          }                              
+        }
+        // Apply default priority if no priority
+        if (hasPriority == false && milestoneTitle != 'New FRs' && 
+          milestoneTitle != '3P Implementation' && milestoneTitle != 'Pending Triage' 
+          && milestone != null) { 
+          updates.push(applyLabel(issue, 'P2: Soon')); 
+        }
+        return Promise.all(updates);
+      });
+    });
+    assert(Array.isArray(issues), `issues must be an array.`);
+    return issues;
+  });	
+}
+
+function errHandler(err) {
+  var msg = err;
+  if (err.message) {
+    msg = err.message;
+  }
+  util.log(util.colors.red(msg));
+}
+
+function applyMilestone(issue, milestoneNumber) {
```

Comment:
I'd recommend documenting every high level function in this file like so:

```
/**
 * Does something to foo and returns a Bar
 *
 * @param {!Foo} foo
 * @return {!Bar}
 */
```

Reply:
done